emsdk.py

```diff
@@ -142,6 +142,9 @@ def exit_with_error(msg):
   ARCH = 'x86'
 elif machine.startswith('aarch64') or machine.lower().startswith('arm64'):
   ARCH = 'aarch64'
+  # no support for Windows on Arm yet, fallback to x64
+  if WINDOWS:
```

Comment:
Can you print a warning here (using the `errlog` function)?

Reply:
Also, perhaps we should open a bug about missing windows arm64 version of emsdk and reference the bug here?

Reply:
> Can you print a warning here (using the `errlog` function)?

That's better yes! (added in latest version)

Reply:
> 

Yes sure, I just wanted to know first if that workaround would be acceptable.

Reply:
I think you folks are only current users we have on arm64 windows so its mostly up to you if you want to consider this is long or short term solution.  What is the typical performance overhead of x86 emulation?